cluster/log-dump/log-dump.sh

```diff
@@ -503,13 +503,19 @@ function dump_nodes_with_logexporter() {
   local -r cloud_provider="${KUBERNETES_PROVIDER}"
   local -r enable_hollow_node_logs="${ENABLE_HOLLOW_NODE_LOGS:-false}"
   local -r logexport_sleep_seconds="$(( 90 + NUM_NODES / 3 ))"
+  local -r multizonal_logexporter="${MULTIZONAL_LOGEXPORTER:-}"
```

Comment:
I think it is rather a zonal_logexporter than multizonal.

@mborsz - WDYT?

Reply:
Two comments:
1. In fact I think we can make it enabled by default. Given that I believe that this label is set in both zonal and regional clusters, in all cloud providers, I don't think it makes sense to require explicit enabling of this feature
2. Nit: We can skip creating additional variable by using ZONE_NODE_SELECTOR_LABEL for this purpose:
  * setting ZONE_NODE_SELECTOR_LABEL="topology.kubernetes.io/zone" by default
  * if ZONE_NODE_SELECTOR_LABEL="", then skipping setting nodeSelector at all.

Reply:
Re naming multizonal_logexporter variable, how about simply: "ZONE_NODE_SELECTOR_ENABLED"?

Reply:
ZONE_NODE_SELECTOR_ENABLED is fine by me

As for making this enabled by default, I do like that idea, as it could have potentially some side-effects. However I do not have a strong opinion about it.

Reply:
After some thought I'm also leaning towards enabling it by default. The logic already assumes we're using GCP-specific stuff here (using `GOOGLE_APPLICATION_CREDENTIALS` env var in line 502 or even leveraging GCS in [logexporter](https://github.com/kubernetes/test-infra/blob/master/logexporter/cmd/main.go)), hence I believe no other cloud provider actually uses that part of the script.